azure-pipeline.release.yml

```diff
@@ -1,73 +1,53 @@
-trigger: none
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
```

Comment:
When doing patch releases, we release from `release/major.minor` branches. Ex: `release/0.34`.

Reply:
Do we want CI to run on the release branches as well on each commit? Makes sense to me.

Reply:
I see now. This just means that it will only run automatically. I can still manually run on release branches. That's fine, no need to change.